server/routes/api/users.js

```diff
@@ -150,4 +153,32 @@ router.get("/submissions", auth, (req, res) => {
     });
 });
 
-module.exports = router;
\ No newline at end of file
+// @route   GET api/users/:name/
+// @desc    Get users with given name
+// @access  Private
+
+router.get("/:name", auth, (req, res) => {
```

Comment:
Do we think this should be sent via query param instead? `/api/users?name=NAME`. I feel that this would be more standard. Otherwise everything looks really good here, pretty cool!

Reply:
hmm...I just read a document on this and we should be using query param when resource is not specific. I'll update this. Thanks, I am getting to know cool things:)